test/common/performance/sender/http_load_generator.go

```diff
@@ -63,20 +64,27 @@ func NewCloudEventsTargeter(sinkUrl string, msgSize uint, eventType string, even
 }
 
 func (cet CloudEventsTargeter) VegetaTargeter() vegeta.Targeter {
+	body := []byte("\"" + generateRandString(cet.msgSize) + "\"")
+	uuidGen := fastuuid.MustNewGenerator()
+
+	ceType := []string{cet.eventType}
+	ceSource := []string{cet.eventSource}
+	ceSpecVersion := []string{"0.2"}
+	ceContentType := []string{"application/json"}
+
 	return func(t *vegeta.Target) error {
 		t.Method = http.MethodPost
 		t.URL = cet.sinkUrl
 
-		t.Header = make(http.Header)
-
-		t.Header.Set("Ce-Id", uuid.New().String())
-		t.Header.Set("Ce-Type", cet.eventType)
-		t.Header.Set("Ce-Source", cet.eventSource)
-		t.Header.Set("Ce-Specversion", "0.2")
+		t.Header = make(http.Header, 5)
 
-		t.Header.Set("Content-Type", "application/json")
+		t.Header["Ce-Id"] = []string{uuidGen.Hex128()}
```

Comment:
For the sake of consistency, can we also initialize `ceID` in the above code?

Reply:
No because this must be different for each request, while the others can be the same